docs/reference/text-structure/apis/find-message-structure.asciidoc

```diff
@@ -0,0 +1,292 @@
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,038][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [repository-url]",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,042][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [rest-root]",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,043][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [ingest-user-agent]",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,043][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [x-pack-core]tive methods for [MacOS]",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,043][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [x-pack-redact]API enabled; uses preferredBitSize=128",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,044][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [lang-painless]]",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,044][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [repository-s3]",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,044][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [x-pack-analytics]",
+          "count" : 1
+        },
+        {
+          "value" : "[2024-03-05T10:52:41,044][INFO ][o.e.p.PluginsService     ] [laptop] loaded module [x-pack-autoscaling]",
+          "count" : 1
+        }
+      ]
+    },
+    "timestamp" : {
+      "count" : 22,
+      "cardinality" : 14,
+      "earliest" : "2024-03-05T10:52:36,256",
+      "latest" : "2024-03-05T10:52:49,199",
+      "top_hits" : [
+        {
+          "value" : "2024-03-05T10:52:41,044",
+          "count" : 6
+        },
+        {
+          "value" : "2024-03-05T10:52:41,043",
+          "count" : 3
+        },
+        {
+          "value" : "2024-03-05T10:52:41,059",
+          "count" : 2
+        },
+        {
+          "value" : "2024-03-05T10:52:36,256",
+          "count" : 1
+        },
+        {
+          "value" : "2024-03-05T10:52:41,038",
+          "count" : 1
+        },
+        {
+          "value" : "2024-03-05T10:52:41,042",
+          "count" : 1
+        },
+        {
+          "value" : "2024-03-05T10:52:43,291",
+          "count" : 1
+        },
+        {
+          "value" : "2024-03-05T10:52:46,098",
+          "count" : 1
+        },
+        {
+          "value" : "2024-03-05T10:52:47,227",
+          "count" : 1
+        },
+        {
+          "value" : "2024-03-05T10:52:47,259",
+          "count" : 1
+        }
+      ]
+    }
+  }
+}
+----
+// TESTRESPONSE[s/"sample_start" : ".*",/"sample_start" : "$body.sample_start",/]
+// The substitution is because the text is pre-processed by the test harness,
+// so the fields may get reordered in the JSON the endpoint sees
```

Comment:
```suggestion
// TESTRESPONSE
```

In this case I don't see why the test harness would reorder the input. JSON arrays have a well-defined order.